app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/connectors/ThirdPartyApplicationConnector.scala

```diff
@@ -32,19 +32,25 @@ class ThirdPartyApplicationConnector @Inject()(
   httpClient: HttpClient,
   config: ThirdPartyApplicationConnector.Config,
   val metrics: ConnectorMetrics
-)(implicit val ec: ExecutionContext) {
+)(implicit val ec: ExecutionContext) extends ApplicationUpdateFormatters with CommonResponseHandlers {
+
+  import uk.gov.hmrc.http.HttpReads.Implicits._
```

Comment:
I'd pull this to top level now it's needed by more than one fn

Reply:
Done